src/mapping/config/MappingConfiguration.cpp

```diff
@@ -285,19 +285,19 @@ MappingConfiguration::ConfiguredMapping MappingConfiguration::createMapping(
 #endif
 
   bool createPetRBF = false;
+  bool createEigenRBF  = false;
```

Comment:
I don't think an additional variable is required.

You could tidy this function up by introducing an enum though:
```cpp
enum struct RBFType { Eigen, PETSc };
RBFType rbftype = RBFType::Eigen;
// ...
if (rbftype == RBFType::Eigen) {
   // create eigen mapping ...
}
```